src/html/partials/concept.html

```diff
@@ -1,4 +1,73 @@
 <section class="concept-section">
-  <h2 class="sm-title">The Concept</h2>
-  <p class="bg-intro">Meet Florence</p>
+
+  <div class="concept-section__intro">
+    <h2 class="sm-title">The Concept</h2>
+    <p class="bg-intro">Meet Florence</p>
+
+    <p>Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen. A chatbot at heart, patients would converse with her through text and picture messages.</p>
+      
+    <p>If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.</p>
+  </div>
+
+  <ul class="concept-section__list">
```

Comment:
wondering if doing this kind of thing locks you into not making these things reusable if you have something with the same styles but it isn't a "concept-section" ...maybe that's fine though

Reply:
Yeah you're right, but I don't have any other sections with these styles so I figured it would be okay for a small project?